docs/api/SessionPool.md

```diff
@@ -6,7 +6,8 @@ title: SessionPool
 <a name="SessionPool"></a>
 
 Handles the sessions rotation, creation and persistence. Creates a pool of [`Session`](session) instances, that are randomly rotated. When some
-session is marked as blocked. It is removed and new one is created instead.
+session is marked as blocked. It is removed and new one is created instead. You can read more about session management
+[`here`](../guides/sessionmanagement).
```

Comment:
Rather than putting links on "here", it's always better to say what's the name of the article, so that user know where the link leads. E.g. Learn more in the [Session Management Guide](...). Also the longer text makes the link more clickable